api/include/opentelemetry/context/threadlocal_context.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include "opentelemetry/context/context.h"
+#include "opentelemetry/context/runtime_context.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace context
+{
+
+// The ThreadLocalContext class is a derived class from RuntimeContext and
+// provides a wrapper for propogating context through cpp thread locally.
+// This file must be included to use the RuntimeContext class if another
+// implementation has not been registered.
```

Comment:
Just out of interest: do we plan to have any other context handlers besides the `ThreadLocalContext`?

Reply:
I do not believe we are planning on any others.